openmc/mesh.py

```diff
@@ -288,9 +288,16 @@ def build_cells(self, bc=['reflective'] * 6):
                    openmc.YPlane(y0=self.upper_right[1],
                                  boundary_type=bc[3])]
         if twod:
-            zplanes = [openmc.ZPlane(z0=np.finfo(np.float).min,
+            # Would prefer to have the z ranges be the max supported float, but
+            # these values are apparently different between python and Fortran.
+            # Choosing a safe and sane default.
+            # Values of +/-1000 are used here as there seems to be an
+            # inconsistency between what numpy uses as the max float and what
+            # Fortran expects for a real(8), so this avoids code complication
+            # and achieves the same goal.
+            zplanes = [openmc.ZPlane(z0=-1000.,
                                      boundary_type='reflective'),
-                       openmc.ZPlane(z0=np.finfo(np.float).max,
+                       openmc.ZPlane(z0=1000.,
                                      boundary_type='reflective')]
```

Comment:
Can the `boundary_type='reflective')` bits be moved on to the previous lines to clean things up? It looks like the lines should still be under 80 characters.